ledger-core/v2/virtual/execute/execute.go

```diff
@@ -145,6 +145,12 @@ func (s *SMExecute) stepWaitObjectReady(ctx smachine.ExecutionContext) smachine.
 		semaphoreOrdered = state.MutableExecute
 		semaphoreUnordered = state.ImmutableExecute
 
+		if isOrdered {
```

Comment:
Lower, There is following code
if ctx.AcquireForThisStep(semaphoreReadyToWork).IsNotPassed() {
		return ctx.Sleep().ThenRepeat()
}
it means that if we can't acquire the lock we will repeat this step and again increment this numbers

Reply:
Fix multiple increment on retries